src/pages/ReimbursementAccount/ReimbursementAccountPage.tsx

```diff
@@ -116,11 +122,9 @@ const ROUTE_NAMES = {
 /**
  * We can pass stepToOpen in the URL to force which step to show.
  * Mainly needed when user finished the flow in verifying state, and Ops ask them to modify some fields from a specific step.
- * @param {Object} route
- * @returns {String}
  */
-function getStepToOpenFromRouteParams(route) {
-    switch (lodashGet(route, ['params', 'stepToOpen'], '')) {
+function getStepToOpenFromRouteParams(route: RouteProp<ReimbursementAccountNavigatorParamList, typeof SCREENS.REIMBURSEMENT_ACCOUNT_ROOT>): ValueOf<typeof CONST.BANK_ACCOUNT.STEP> | '' {
```

Comment:
You can reuse existing `type BankAccountStep = ValueOf<typeof CONST.BANK_ACCOUNT.STEP>;`